### DISCUSSION TITLE
ArrayTable某个列设置了x-display: hidden为什么该列的字段不在form.values里

### DISCUSSION THREAD

**Discussion by SuperRun** [alibaba/formily#3349] [Q&A 1.x]
Demo
https://codesandbox.io/s/array-table-display-c5u6hg?file=/config.ts

问题描述
ArrayTable某个列设置了x-display: hidden为什么该列的字段不在form.values里
或者对于这种场景(保留字段但是不展示列如何实现)

![image](https://user-images.githubusercontent.com/28804901/185575134-b605872f-ce6a-4c6e-afc1-0e3609b87017.png)

**Comment by janryWang** [comment]
保留字段，不展示列，你不定义这一列不就行了，这种应该属于后端冗余数据，而不是前端冗余数据

**Reply by SuperRun** [reply]
有些时候确实需要传递冗余数据 增加一列可以减少数据处理这一步 想问这种场景能否实现

**Reply by SuperRun** [reply]
文档对于 x-display: hidden 的定义就是保留字段 但是不展示 但是在ArrayTable中使用的效果不符合这个结果
